bokehjs/src/lib/core/util/menus.ts

```diff
@@ -78,14 +92,34 @@ export class ContextMenu {
     window.removeEventListener("blur", this._on_blur)
   }
 
-  protected _position(at: ScreenPoint): void {
+  protected _position(at: At): void {
     const parent_el = this.el.parentElement
     if (parent_el != null) {
+      const pos = (() => {
+        if ("left_of" in at) {
+          const {left, top} = at.left_of.getBoundingClientRect()
+          return {right: left, top}
+        }
+        if ("right_of" in at) {
+          const {top, right} = at.right_of.getBoundingClientRect()
+          return {left: right, top}
+        }
+        if ("below_of" in at) {
+          const {left, bottom} = at.below_of.getBoundingClientRect()
+          return {left, top: bottom}
+        }
+        if ("above_of" in at) {
+          const {left, top} = at.above_of.getBoundingClientRect()
+          return {left, bottom: top}
```

Comment:
"above of" and "below of" are not grammatical, these should just be "above" and "below"